app/models/appointment_slot_preset.rb

```diff
@@ -18,11 +18,11 @@ class AppointmentSlotPreset < ActiveRecord::Base
 
   def no_conflict
     conflicts = AppointmentSlotPreset
-      .where({ day: self.day, hours: self.hours, minutes: self.minutes })
+      .where({ day: day, hours: hours, minutes: minutes })
```

Comment:
How about writing single where query for it?

where(''day = (?) and hours = (?) minutes= (?) and id != (?)" )

Correct me if it will be different apart from syntax?  :)

Reply:
should result in the same thing 👍 

just a preference thing, I find `where(''day = (?) and hours = (?) minutes= (?) and id != (?)" , day, hours, minutes, id)` to be a bit too long

Reply:
long live nosql 😄